docs/cli.md

```diff
@@ -29,17 +34,310 @@ Common Flags
 | --insecure-skip-tls-verify | Skip SSL certificate validation (will make HTTPS insecure) |
 | --help (-h)                | Display help for specified command |
```

Comment:
Put the flags into backticks, so that they are formatted as code. See like we did that in [s2i](https://github.com/openshift/source-to-image/blob/master/docs/cli.md).